### PR TITLE
✨ Hot Module Reloading

### PR DESCRIPTION
**Pull request by bdolly**
Attempting to setup react-hot-loader so there is no page refresh on save.

### CODE REVIEW

**Comment by bdolly** [comment]
@jgnieuwhof got this working by just turning off Webpack dev server page refresh in the `config-overrides`.

File: src/components/SplashPage.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const SplashPage = compose(withTheme)(({ theme, children }) => (
     css={`
       width: 630px;
       margin: auto;
+      border: 3px solid green;
```
Comment (hlminh2000) [inline src/components/SplashPage.js]: 🔥

**Comment by bdolly** [comment]
@hlminh2000 I think this is g2g, not sure what's up with the netlify status blocking it thought

**Comment by hlminh2000** [comment]
@bdolly it was just missing some dependencies from `package.json` :P ran a test local prod build and it works after some installs. Netlify is working now too but not sure why it's not reporting back to github... Anyway, merging now :)

**Pull request merged**